depfinder/reports.py

```diff
@@ -140,28 +102,29 @@ def report_conda_forge_names_from_import_map(total_imports, builtin_modules=None
             futures[future] = md
     for future in as_completed(futures):
```

Comment:
I wonder if we still need the threadpool approach here? seems like now we're just looking up into a dictionary. in a follow-on PR could probably drop this complexity

Reply:
We do. The depfinder code doesn't know what the conda-forge-metadata package is doing underneath. It could be making a bunch of http requests or calls to a DB endpoint.